### DISCUSSION TITLE
Decreasing the growth rate of dendrites using DiscreteNucleation material property

### DISCUSSION THREAD

**Discussion by vermaprk** [idaholab/moose#25364] [Q&A Modules: Phase field] [answered]
Hello

I have a phase field model depicting electro deposition in a non-uniform way as shown in figure (attached). However, I am trying to stop the growth at the spikes by manipulating the phase-field variable mu (grand potential PFM). I am using DiscreteNucleation material property to increase the magnitude of mu at the interface of the spike so that the growth takes place only at the flat area rather than the spike. This should result in a overall much smoother/flatter profile. 

However, I am not able to achieve this using DiscreteNucleation material property which is trying to force mu to -20. A high (-ve) value of mu should result in lower concentration at the peak causing lower growth rate at spike as compared to the flatter region.

Thanks
@dschwen @laagesen @GiudGiud 

![Screenshot 2023-08-31 084155](https://github.com/idaholab/moose/assets/69255835/b11510f9-2c9c-4068-90df-4ac106e88cac)

**Comment by laagesen** [comment]
Hi, I'm not sure why you're trying to stop the growth at the spikes. It seems like if this a natural feature that emerges from the physics of the model, you should not try to artificially suppress it. Dendrite formation can certainly happen in electrodeposition. Can you explain why you are trying to suppress growth?

**Comment by vermaprk** [comment]
Yes, I am trying to model the effect of additives which will preferentially
be adsorbed at the spikes because of higher electric field near them.
Further, these additives will repel the metal ion and lowering their concentration near the
spike interface leading to suppressed growth.

I am referring to this article
https://doi.org/10.1021/ja312241y

![Screenshot_20230902-070723-620.png](https://github.com/idaholab/moose/assets/69255835/c9d5e69b-1d46-45d1-82bd-9441512723fb)

**Reply by laagesen** [reply, marked as answer]
I am not an expert in this area, but I don't think nucleation is the right way to go about suppressing growth here. This is a pretty complicated physical problem. Off the top of my head it seems that you would need to add a coupled electric field and solve Poisson's equation to understand the effect of additive concentration on electric field, and then the resulting field would be what is responsible for pushing the metal ions away. I'd suggest to do some literature search to find an existing phase-field model for this problem and we can try to help you implement that once you find it.

**Reply by vermaprk** [reply]
OK. Thanks for the help. I will certainly get back to you.
